test/stream/bridge/keboola/keboola_test.go

```diff
@@ -25,9 +29,7 @@ import (
 )
 
 // To see details run: TEST_VERBOSE=true go test ./test/stream/bridge/... -v.
-func TestKeboolaBridgeWorkflow(t *testing.T) {
-	t.Parallel()
-
+func TestKeboolaBridgeWorkflow(t *testing.T) { // nolint: paralleltest
```

Comment:
why no parallel?

Reply:
There is issue in etcd connection when being in parallel. Looks like we have some issue in the test suite. I will create subsequent tasks to dig into the issue. Simply adding parallel will make the test fail in CI for sure. The result locally varies.

Reply:
https://keboola.atlassian.net/browse/PSGO-972